MathLib/LinAlg/BLAS.cpp

```diff
@@ -20,13 +20,29 @@ namespace MathLib { namespace BLAS
 {
 
 // Explicit specialization
+// Computes the Manhattan norm of x
```

Comment:
In numerics it is usually called p1 norm (or more correctly l1 (like small-ell-one, $\ell_1$) norm). There are small-ell spaces and L-spaces with their p-norms... But Manhatten is not wrong, just unusual to me.